go.mod

```diff
@@ -0,0 +1,27 @@
+module github.com/threefoldtech/tfgateway
+
+go 1.14
+
+replace github.com/threefoldtech/zos => ../zos
+
+replace github.com/threefoldtech/tfexplorer => ../tfexplorer
```

Comment:
Are we sure we want local dependencies?

Reply:
we don't but I need a version released on tfexplore first.